numba/tests/test_bytes.py

```diff
@@ -0,0 +1,41 @@
+from numba.tests.support import (TestCase, no_pyobj_flags, MemoryLeakMixin)
+from numba import njit
+import unittest
+
+
+def addbytes_len_usecase(a, b):
+    output = len(b''.join([a, b]))
```

Comment:
I presume this slightly unusual spelling of `a + b` is due to these bugs: #7174 #7921 ?